os/arch/arm/src/s5j/s5j_dma.h

```diff
@@ -262,68 +262,68 @@ static inline int DMA_Encode_DMALP(char *base, DMA_LC lc, void **iter)
 	return 2;
 }
 
-static inline int DMA_Encode_DMALPEND(char *base, u8 CTRL, char backwards_jump)
+inline static int DMA_Encode_DMALPEND(char *base, u8 CTRL, char backwards_jump)
 {
 	base[0] = 0x28 | CTRL;
 	base[1] = backwards_jump;
 	return 2;
 }
 
-static inline int DMA_Encode_DMALD(char *base, u8 CTRL)
+inline static int DMA_Encode_DMALD(char *base, u8 CTRL)
 {
 	base[0] = 0x04 | CTRL;
 	return 1;
 }
 
-static inline int DMA_Encode_DMALDP(char *base, u8 CTRL, DMA_REQ_MAP periph)
+inline static int DMA_Encode_DMALDP(char *base, u8 CTRL, DMA_REQ_MAP periph)
 {
 	base[0] = 0x25 | CTRL;
 	base[1] = periph << 3;
 	return 2;
 }
 
-static inline int DMA_Encode_DMAST(char *base, u8 CTRL)
+inline static int DMA_Encode_DMAST(char *base, u8 CTRL)
 {
 	base[0] = 0x08 | CTRL;
 	return 1;
 }
 
-static inline int DMA_Encode_DMASTP(char *base, u8 CTRL, DMA_REQ_MAP periph)
+inline static int DMA_Encode_DMASTP(char *base, u8 CTRL, DMA_REQ_MAP periph)
 {
 	base[0] = 0x29 | CTRL;
 	base[1] = periph << 3;
 	return 2;
 }
 
-static inline int DMA_Encode_DMAWFP(char *base, u8 CTRL, DMA_REQ_MAP periph)
+inline static int DMA_Encode_DMAWFP(char *base, u8 CTRL, DMA_REQ_MAP periph)
 {
 	base[0] = 0x30 | CTRL;
 	base[1] = periph << 3;
 	return 2;
 }
 
-static inline int DMA_Encode_DMASEV(char *base, void **event_num)
+inline static int DMA_Encode_DMASEV(char *base, void **event_num)
 {
 	base[0] = 0x34;
 	base[1] = 0xe;
 	*event_num = base + 1;
 	return 2;
 }
 
-static inline int DMA_Encode_DMAEND(char *base)
+inline static int DMA_Encode_DMAEND(char *base)
 {
 	base[0] = 0x00;
 	return 1;
 }
 
-static inline int DMA_Encode_DMAFLUSHP(char *base, u8 periph)
+inline static int DMA_Encode_DMAFLUSHP(char *base, u8 periph)
 {
 	base[0] = 0x35;
 	base[1] = periph << 3;
 	return 2;
 }
 
-static inline int DMA_Encode_DMAWMB(char *base)
+inline static int DMA_Encode_DMAWMB(char *base)
```

Comment:
Revert this file. We fixed a problem. But you revert and make a problem. Why dont you check latest codes?

Reply:
Restored